.github/workflows/publish-action.yml

```diff
@@ -0,0 +1,27 @@
+name: publish-action
+on:
+  release:
+    types: released
+concurrency: ${{ github.workflow }}
+jobs:
+  publish-action:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          cache: npm
+      - run: npm ci
+      - run: npm run build
+      - run: rm -rf node_modules
+      - uses: actions4git/setup-git@v1
```

Comment:
it essentially does this:

```sh
git config --global user.name "github-actions[bot]"
git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

so that any future `git commit` doesn't error with a "tell me who you are" message